homeassistant/components/refoss/strings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "config": {
+    "step": {},
```

Comment:
Can this be removed as well?

Reply:
aaah right

Reply:
after removed, pre-commit:  Invalid strings.json: required key not provided @ data['config']['step']. Got None